Client/game.js

```diff
@@ -466,6 +466,8 @@ socket.on('take completed stories', function (stories, numStages, commStrokes =
             } else {
                 scrollHtml += s.owners[j - 1] + ' drew:<br>';
                 scrollHtml += '<img width="480" height="384" class="art" src="' + strokesToDataUrl(s.images[idx]) + '">';
+				scrollHtml += '<br> <img class="like-button" src="like_off.png" id="like ' + i + ' ' + j + '" onclick="likeImage(\'like ' + i + ' ' + j + '\')">';
+				scrollHtml += '<span class="like-counter" id="' + 'counter ' + i + " " + j + '">0</span> <br>';
```

Comment:
Couldn't `id="' + 'counter ' +` just be `id="counter ' +` ?

Reply:
oh yeah lol